src/components/CCPDashboard/PatientInfoTable.tsx

```diff
@@ -230,153 +306,106 @@ export const PatientInfoTable = ({
-            width="104px"
-            style={{ maxWidth: '104px' }}
-          >
-            {statusLabels[patient.status]}
-          </TableCell>
-          {type === CCPDashboardTabOptions.PatientOverview && (
-            <>
-              <TableCell
-                className={classes.tableCell}
-                width="128px"
-                style={{ maxWidth: '128px' }}
-              >
-                {patient.hospitalId?.name}
-              </TableCell>
-              <TableCell
-                className={classes.tableCell}
-                width="192px"
-                style={{ maxWidth: '192px' }}
-              >
-                {moment(patient.updatedAt).format('MMM D YYYY, h:mm A')}
-              </TableCell>
-            </>
-          )}
-          {type === CCPDashboardTabOptions.Hospital && (
-            <>
-              <TableCell
-                className={classes.tableCell}
-                width="132px"
-                style={{ maxWidth: '132px' }}
-              >
-                {patient.runNumber}
-              </TableCell>
-              <TableCell
-                className={classes.tableCell}
-                width="192px"
-                style={{ maxWidth: '192px' }}
-              >
-                {moment(patient.transportTime).format('MMM D YYYY, h:mm A')}
-              </TableCell>
-            </>
-          )}
-          <TableCell width="48px" style={{ maxWidth: '48px' }}>
-            <IconButton
-              color="inherit"
-              onClick={(e) => handleClickOptions(e, patient)}
-            >
-              <MoreHoriz />
-            </IconButton>
-          </TableCell>
-        </TableRow>
+  const handleClickRestore = () => {
+    setAnchorElRestore(null);
+    setOpenRestorePatient(true);
+  };
+
+  const handleRestoreDialogBtn = () => {
+    if (
+      selectedPatient !== undefined &&
+      selectedPatient.status === Status.DELETED
+    ) {
+      setOpenRestorePatient(true);
+    } else {
+      history.push(
+        `/events/${eventId}/ccps/${ccpId}/patients/${
+          ((selectedPatient as unknown) as Patient).id
+        }`
       );
     }
-  );
+  };
+
+  const getTableRows = (isActive: Boolean, patients: Patient[]) => {
+    return stableSort(patients, getComparator(order, orderBy)).map(
+      (patient: Patient) => {
+        const style = isActive ? classes.tableCell : classes.deletedTableCell;
+        const renderTableCell = (style, width, value) => {
+          let content = patient[value];
+          let border = {};
+
+          switch (value) {
+            case 'triageLevel':
+              content = triageLevels[content].label;
+              border = {
+                borderLeft: `${isActive ? '16px' : '0px'} solid ${
+                  triageLevels[patient.triageLevel].colour
+                }`,
+              };
+              break;
+            case 'hospitalId.name':
+              content = patient.hospitalId.name;
```

Comment:
I'm getting a `TypeError: Cannot read property 'name' of null`
```suggestion
              content = patient.hospitalId?.name;
```